test/functional/core/channels_spec.lua

```diff
@@ -90,10 +90,28 @@ describe('channels', function()
     eq({"notification", "stdout", {id, {"[1, ['howdy'], 'stdin']"}}}, next_msg())
 
     command("call chanclose(id, 'stdin')")
-    expect_twostreams({{"notification", "stdout", {id, {"[1, [''], 'stdin']"}}},
-                       {'notification', 'stdout', {id, {''}}}},
-                      {{"notification", "stderr", {id, {"*dies*"}}},
-                       {'notification', 'stderr', {id, {''}}}})
+    expect_msg_seq(
+      { {"notification", "stdout", {id, {"[1, [''], 'stdin']"}}},
+        {'notification', 'stdout', {id, {''}}},
+        {"notification", "stderr", {id, {"*dies*"}}},
+        {'notification', 'stderr', {id, {''}}},
+      },
+      { {"notification", "stdout", {id, {"[1, [''], 'stdin']"}}},
+        {'notification', 'stdout', {id, {''}}},
+        {'notification', 'stderr', {id, {''}}},
+        {"notification", "stderr", {id, {"*dies*"}}},
```

Comment:
This manually-specified permutation list is annoying and verbose. Might try to do this programmatically.